### PR TITLE
Issue 777 fix

### PR DESCRIPTION
**Pull request by rg666**
Fix for issue [777](https://github.com/cobrateam/splinter/issues/777): Provided a  "get" method to access the HTML attributes' element list.

Test method test_method_get added to tests\test_element_list.py and tested successfully.

### CODE REVIEW

File: splinter/driver/webdriver/__init__.py [hunk 1]
```diff
@@ -748,6 +748,9 @@ def __init__(self, element, parent):
     def _get_value(self):
         return self["value"] or self._element.text
 
+    def get(self,value):
```
Comment (jsfehler) [inline splinter/driver/webdriver/__init__.py:751]: This is incorrect. The implementation of getting with a list index must be done through `__getitem__`

File: tests/test_element_list.py [hunk 2]
```diff
@@ -102,3 +103,15 @@ def test_not_found_exception_with_query_and_method(self):
         expected_message = 'no elements could be found with id "menu"'
         e = cm.exception
         self.assertEqual(expected_message, e.args[0])
+
+    def test_method_get(self):
+        """
+        should provide a \"get\" method
+        which provides access to HTML attributes' element list of the visited webpage
+        """
+        wd_ins = splinter.Browser()
```
Comment (jsfehler) [inline tests/test_element_list.py:112]: You don't need a live browser instance to test getting attributes. You can just use a WebDriverElement instance.

**Review by jsfehler — CHANGES_REQUESTED** [review]

File: splinter/driver/webdriver/__init__.py (hunk 1 above)
Comment (jsfehler) [reply splinter/driver/webdriver/__init__.py:751]: To clarify:

I don't see a reason to add a `get` method.

However, if you were, you'd need to add it to the ElementAPI, which is the blueprint for WebDriverElement, and then support it across all the Elements, not just WebDriver.

Returning `self[value]` isn't what #777 was about. You would want to return the html attributes.

Comment (rg666) [reply splinter/driver/webdriver/__init__.py:751]: Valid point, I'll work towards the review points.

**Comment by rg666** [comment]
Closing this PR.

**Pull request closed without merging**